site/docs/src/json/scim/scim-extended-error-response.json

```diff
@@ -0,0 +1,24 @@
+{
+  "detail": "Request failed, see errors for additional details.",
+  "io.fusionauth.errors": {
```

Comment:
`urn:ietf:params:scim:schemas:extension:fusionauth:2.0:Error` and other places in this PR

Reply:
Oh, I forgot we updated that. I'll fix it.

Reply:
done